canopen/lss.py

```diff
@@ -132,14 +132,15 @@ def send_switch_state_selective(self,
 
         self.__send_lss_address(CS_SWITCH_STATE_SELECTIVE_VENDOR_ID, vendorId)
         self.__send_lss_address(CS_SWITCH_STATE_SELECTIVE_PRODUCT_CODE, productCode)
-        self.__send_lss_address(CS_SWITCH_STATE_SELECTIVE_REVISION_NUMBER, revisionNumber)
-        response = self.__send_lss_address(CS_SWITCH_STATE_SELECTIVE_SERIAL_NUMBER, serialNumber)
+        self.__send_lss_address(
+            CS_SWITCH_STATE_SELECTIVE_REVISION_NUMBER, revisionNumber
```

Comment:
Definitely need to agree on a line length setting for Black.  Changes such as this one cause unnecessary churn and I don't think it's really solving a problem.  My usual compromise is to set a 96 character limit.  Not quite conservative (< 80), but also not as disruptive as allowing 120 chars (+50 %) and it leaves some margin for diff markings.